framework/assets/yii.js

```diff
@@ -279,10 +279,10 @@ window.yii = (function ($) {
             for (var i = 0, len = pairs.length; i < len; i++) {
                 var pair = pairs[i].split('=');
                 var name = decodeURIComponent(pair[0].replace(/\+/g, '%20'));
-                var value = decodeURIComponent(pair[1].replace(/\+/g, '%20'));
                 if (!name.length) {
                     continue;
                 }
+                var value = decodeURIComponent(pair[1].replace(/\+/g, '%20'));
```

Comment:
I think this should be fixed differently. In error case we have an array with one element - empty string - `[""]`. It's better to check and remove it from array before the loop (or construct array in a way that prevents it) so it will not be considered as pair at all. We don't have any name - value pairs with url `/posts/index?`, I think it's more correct.